kube-hunter.py

```diff
@@ -39,6 +40,14 @@
 else:
     config.reporter = PlainReporter()
 
+from src.modules.dispatchers.stdout import STDOUTDispatcher
+from src.modules.dispatchers.http import HTTPDispatcher
+dispatchers = {
+    'stdout': STDOUTDispatcher(),
+    'http': HTTPDispatcher()
+}
+config.dispatcher = dispatchers[config.dispatch.lower()]
```

Comment:
That's a great method for dynamically determine which class to use. 
But it would be better if you instantiate the class after the dictionary access, instead of at the dictionary definition.
it would look something like:
```python
dispatchers = {
    'stdout': STDOUTDispatcher,
    'http': HTTPDispatcher
}
config.dispatcher = dispatchers[config.dispatch.lower()]()
```
the idea was to use this to set the reporter as well, so you can definitely change that as well, if you like to :)

Reply:
good spot, done as suggested.

Reply:
applied to reporters and also caught a condition where an invalid reporter/dispatcher is selected (the 'else" case).